src/gzchunked/mod.rs

```diff
@@ -99,16 +116,76 @@ mod tests {
     use super::*;
     use rand::{Rng, SeedableRng, rngs::StdRng};
 
+    #[test]
+    fn test_encode_and_decode_empty() {
+        let mut encoder = GzChunkedEncoder::new(GzChunkedCompression::default());
+        let mut decoder = GzChunkedDecoder::new();
+
+        let encoded_block = encoder.next_chunk().unwrap();
+        // should contain gzip header
```

Comment:
Comments should be proper written text (start with uppercase letter and end with a dot).